docs/tests.md

```diff
@@ -2,6 +2,11 @@
 
 If you want to run the unit tests for `mass2` go to your `mass2` directory and do `pytest .`
 
-If you want to add tests to `mass2`, you do _not_ need to use the old `unittest` framework. Simpler, more modern tests in the `pytest` style are allowed (and preferred). Put any new tests in a file somewhere inside `tests` with a name like `test_myfeature.py`, it must match the pattern `test_*.py` to be found by pytest.
+If you want to add tests to `mass2`, you do _not_ need to use the old `unittest` framework. Simpler, more modern tests in the `pytest` style are allowed (and preferred). Put any new tests in a file somewhere inside `tests` with a name like `test_myfeature.py`, it must match the pattern `test_*.py` to be found by pytest. Use assertions, `numpy.allclose`, and similar tools to test that outcomes match expectations.
```

Comment:
I wouldn't mention `unittest` at all, just link to `pytest` / explain how to use `pytest`.